app/routes/homepage.jsx

```diff
@@ -62,6 +64,35 @@ export default function HomePage() {
   }, [tokenReady, dataFetched, api]);
 
 
+  useEffect(() => {
```

Comment:
This UseEffect has the same functions as the UseEffect at "NotificationsDropdown.jsx"
Is it possible that there is redundancy?

Reply:
Thanks for commenting on that. In this specific case it is necessary to make the request from homepage and from notisDropdown. It is not redundancy or a code smell (duplicated code) without reason, because we need the request to be triggered in notisDropdown each time the component it is visible.